_docker/drupal/scripts/drupal_install_checker.rb

```diff
@@ -0,0 +1,71 @@
+require 'open3'
+require 'fileutils'
+
+class ProcessExecutor
+  def exec!(cmd, args = [])
+    out, status = Open3.capture2e(cmd, *args)
+    raise out if status.exitstatus != 0
+    out
+  end
+end
+
+class DrupalInstallChecker
+  attr_reader :drupal_site, :process_executor
+
+  def initialize(drupal_site, process_executor = ProcessExecutor.new)
+    @drupal_site = drupal_site
+    @process_executor = process_executor
+  end
+
+  def settings_exists?
+    File.exists? File.join drupal_site, 'settings.php'
+  end
+
+  def rhd_settings_exists?
+    File.exists? File.join drupal_site, 'rhd.settings.php'
+  end
+
+  def mysql_connect?
+    begin
+      process_executor.exec! 'mysql','drupal'
```

Comment:
Same comment as below about this needing to be configurable on a per-environment basis.